packages/shortest/src/commands/init/index.ts

```diff
@@ -0,0 +1,124 @@
+import { execSync } from "child_process";
+import { readFile, writeFile } from "node:fs/promises";
+import { join } from "path";
+import { fileURLToPath } from "url";
+import { detect, resolveCommand } from "package-manager-detector";
+import pc from "picocolors";
+import { CONFIG_FILENAME, ENV_LOCAL_FILENAME } from "../../constants";
+import { addToEnv } from "../../utils/add-to-env";
+import { addToGitignore } from "../../utils/add-to-gitignore";
+
+export default async function main() {
+  console.log(pc.blue("Setting up Shortest..."));
+
+  try {
+    const packageJson = await getPackageJson();
+    if (
+      packageJson?.dependencies?.["@antiwork/shortest"] ||
+      packageJson?.devDependencies?.["@antiwork/shortest"]
+    ) {
+      console.log(pc.green("✔ Package already installed"));
+      return;
```

Comment:
Important to exit early if the package is already installed, otherwise there would be some considerable effort to ensure the install doesn't overwrite any existing data.